spec/features/managing_users_spec.rb

```diff
@@ -98,6 +98,30 @@
       expect(page).to have_text "admin@example.com" # ...with user data
     end
 
+    scenario "I see user pledging history on the user page" do
+      @user    = create(:user)
+      item    = create(:item)
+      # pledge  = user.pledges.create!(item_id: item.id)
+      wishlist      = Wishlist.create!(name: "Charity")
+      item          = Item.create!(name: "Puppy", price_cents: 50)
+      wishlist_item = wishlist.wishlist_items.create!(item_id: item.id)
+
+      pledge_one    = @user.pledges.create!(wishlist_item_id: wishlist_item.id)
+      pledge_two    = @user.pledges.create!(wishlist_item_id: wishlist_item.id)
+
+      @pledges = @user.pledges
+      #byebug
+
+      click_link "Users"
+      users = find_all(".user")
+      within(users.last) do
+        click_link "Show"
+      end
+
+      expect(page).to have_text(pledge_one.wishlist_item.item.name)
+    end
+
+    # scenario "I can assign a user to be Site Manager of a wishlist"
```

Comment:
I just want to make sure I'm deleting the right line; you're wanting line 124 or line 121 deleted?